packages/webpack-config-single-spa/package.json

```diff
@@ -24,9 +24,7 @@
   "dependencies": {
     "clean-webpack-plugin": "^3.0.0",
     "unused-files-webpack-plugin": "^3.4.0",
-    "webpack-bundle-analyzer": "^3.6.0"
-  },
-  "peerDependencies": {
+    "webpack-bundle-analyzer": "^3.6.0",
     "babel-loader": "*",
```

Comment:
Let's specify exact version now, instead of `*`? The star was to try to avoid situations where the user-provided version was incorrect and that resulting in a broken build. However, now that the user doesn't have to provide the version anymore, I don't think that's a problem?

Reply:
Good call. I'll add the exact versions.